libnavigation-base/src/main/java/com/mapbox/navigation/base/route/NavigationRoute.kt

```diff
@@ -135,6 +135,8 @@ class NavigationRoute internal constructor(
          *
          * @param directionsResponseJson response to be parsed into [NavigationRoute]s
          * @param routeRequestUrl URL used to generate the [directionsResponse]
+         *
+         * @throws Throwable if `directionsResponseJson` is invalid
```

Comment:
What type of Throwable it throws? Can it ever be `Error`, not `Exception`?

Reply:
It's supposed to be exception I guess, but it all comes down to gson library, so I don't want to promise anything in their name.